### PR TITLE
[MM-48027] improve use case window style

### PR DESCRIPTION
**Pull request by mhd-sln**
#### Summary
For a high level understanding of the feature. Please view the parent PR. This PR improves the styling of use-case window based on https://www.figma.com/proto/xS2v61Rlq2DiLG8QF3fIPU/MMO-MVP?node-id=200%3A159143&scaling=min-zoom&page-id=193%3A104002&starting-point-node-id=243%3A163187

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48027

#### Video
https://www.loom.com/share/f762eadbcc7641bea820be8c631d42db

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by JulienTant — COMMENTED** [review]
This is magic to me! 🤯 

Only issue I found was the padding around the text seems off

image

File: components/worktemplates/components/menu/use_case.tsx [hunk 1]
```diff
@@ -100,63 +102,86 @@ const StyledUseCaseMenuItem = styled(UseCase)`
...
 
         img {
             width: 204px;
             height: 123px;
+            z-index: 1;
+            transition: margin 0.4s ease-in-out;
         }
     }
 
     .name {
```
Comment (julmondragon) [inline components/worktemplates/components/menu/use_case.tsx:136]: Please make sure to match styles for this section from figma design in all the states: https://www.figma.com/file/xS2v61Rlq2DiLG8QF3fIPU/MMO-MVP?node-id=592%3A194495

**Review by julmondragon — COMMENTED** [review]
Looks good!

File: components/worktemplates/components/menu/use_case.tsx [hunk 2]
```diff
@@ -99,62 +104,82 @@ const StyledUseCaseMenuItem = styled(UseCase)`
...
         .details {
-            display: none;
+            visibility: hidden;
+            margin-bottom: 12px;
+            opacity: 0;
             font-weight: 400;
             font-size: 11px;
             line-height: 16px;
             letter-spacing: 0.02em;
             color: rgba(var(--center-channel-color), 0.72);
```
Comment (julmondragon) [inline components/worktemplates/components/menu/use_case.tsx:154]:
```suggestion
            color: rgba(var(--center-channel-text-rgb), 0.72);
```

**Review by julmondragon — CHANGES_REQUESTED** [review]

**Review by JulienTant — APPROVED** [review]

**Review by julmondragon — APPROVED** [review]
Thanks @mhd-sln !

**Review by stevemudie — APPROVED** [review]
This looks and works _really_ good. Nice work @mhd-sln!

**Comment by mhd-sln** [comment]
Discussed this with @JulienTant and we thought that it might be a good idea to merge this and https://github.com/mattermost/mattermost-webapp/pull/11557 into https://github.com/mattermost/mattermost-webapp/pull/11475.
We would be revisiting the code from UX review perspective later as well once the Work Template Modal would be feature ready in the future. @annelieseklein

**Pull request merged**